buildprocess/webpack.config.js

```diff
@@ -5,11 +5,18 @@ const configureWebpackForTerriaJS = require("terriajs/buildprocess/configureWebp
 const configureWebpackForPlugins = require("./configureWebpackForPlugins");
 const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 const path = require("path");
+const HtmlPlugin = require("html-webpack-plugin");
+var minimist = require("minimist");
 
 /**
  * Webpack config for building terriamap
  */
 module.exports = function (devMode) {
+  var args = minimist(process.argv.slice(2), {
+    string: ["baseHref"],
+    default: { baseHref: "/" }
+  });
```

Comment:
I think it is better to be explicit here and pass these in as parameters instead of reading directly from `argv`.

Reply:
Move the argv processing to gulpfile